packages/strapi-plugin-content-manager/controllers/single-types.js

```diff
@@ -80,14 +83,15 @@ module.exports = {
         return ctx.forbidden();
       }
 
-      const updatedEntity = await entityManager.update(entity, sanitizeFn(body), model);
+      const updatedEntity = await entityManager.update(query, sanitizeFn(body), model);
```

Comment:
once you know what entity to update why do you need to use query to do the update ? Same comment as before It looks like you can just pass the entity & the entity manager will do the rest ?

Reply:
If was because the locale params was automatically added.
I made a proposal to handle it in the decorator